Formula/ansible.rb

```diff
@@ -59,40 +60,24 @@ class Ansible < Formula
   end
   ### end
 
-  resource "Babel" do
```

Comment:
in particular! some of these look like they are old and are not sorted; if those end up sorted, that is good. IIRC poet sorts "Banana" before "apple", so initial-caps should stay at the tops, but maybe I'm wrong, or confused.